web_src/js/features/repo-legacy.js

```diff
@@ -429,7 +432,7 @@ async function onEditContent(event) {
         content: comboMarkdownEditor.value(),
         context: editContentZone.getAttribute('data-context'),
       });
-      for (const file of dz.files) params.append('files[]', file.uuid);
+      for (const fileInput of dropzoneInst?.element.querySelectorAll('.files [name=files]')) params.append('files[]', fileInput.value);
```

Comment:
Can we avoid having the query files out of the DOM here? Ideally we should always interface with `dz`, DOM is not means as a reliable data store.

Reply:
No at the moment, because the dropzone was abused. The whole code heavily depends on the `.files`